source/js/components/TomatoSlider.js

```diff
@@ -110,6 +116,38 @@ class TomatoSlider extends HTMLElement {
     this.colorTomatos(n, 'red');
   }
 
+  /**
+   * Change the number of tomatoes in the current slider with Ctr+LeftArrow and Ctr+RightArrow
+   * @param {KeyboardEvent} e
+   */
+  handleKeyDown(e) {
+    this.keysPressed[e.key] = true;
+    if (this.keysPressed.Control && this.keysPressed.ArrowLeft && !e.repeat) {
+      this.input.value = Math.max(this.input.value - 1, 1);
+      this.colorTomatos(this.input.value, 'red');
+    }
+    if (this.keysPressed.Control && this.keysPressed.ArrowRight && !e.repeat) {
+      if (this.input.value === '1') {
```

Comment:
Can we replace these with `this.input.value = Math.min(this.input.value + 1, 5);`?

Reply:
We tried that, but this.input.value is a string, so it concatenates 1 to the value which breaks the functionality. It works for the max value on the left key with subtract, but we weren't sure why.

Reply:
We could cast it with `Number(...)`

Reply:
Have you guys looked into something like https://www.w3schools.com/jsref/jsref_parseint.asp to get a number?